doc/reference/reference_lua/box_events/system_events.rst

```diff
@@ -99,6 +101,26 @@ Contains schema-related data.
     MP_STR “version”: MP_UINT schema_version,
     }
 
+.. _system-events_box-shutdown:
+
+box.shutdown
+~~~~~~~~~~~~
```

Comment:
Even though the graceful shutdown protocol implemented using built-in events, it's still a part of the binary protocol, which should be interesting only to connector developers. I think it's better to describe the graceful shutdown protocol in a [Binary protocol](https://www.tarantool.io/en/doc/latest/dev_guide/internals/box_protocol/) sub-section, and refer to that sub-section here.

Reply:
Moved the shutdown protocol description to Binary protocol page